extensions/amp-video/0.1/validator-amp-video.protoascii

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-video
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "SCRIPT"
+  spec_name: "amp-video extension .js script"
+  mandatory_parent: "HEAD"
+  unique_warning: true
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-video"
+    dispatch_key: true
+  }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-video-(latest|0\\.1)\\.js"
+  }
+  attrs: {
+    name: "type"
+    value_casei: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/extended/amp-video.html"
+}
+tags: {  # <amp-video>
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "AMP-VIDEO"
+  disallowed_ancestor: "AMP-SIDEBAR"
+  also_requires_tag: "amp-video extension .js script"
+  attrs: { name: "autoplay" }
```

Comment:
these look like the amp-youtube attributes. amp-video has a lot more. Let's just copy/paste what's in `validator-main.protoascii` for amp-video.